### ISSUE TITLE
Browsh Docker stuck on "Waiting for Firefox to connect..."

### ISSUE THREAD

**Issue by InconsolableCellist** [browsh-org/browsh#291]
Just like #265:

I built the Docker image of the vim-mode-experimental branch, but it didn't work. I installed with:

```
git clone -b vim-mode-experimental --single-branch https://github.com/browsh-org/browsh.git
cd browsh
docker build -t browshvim .
docker run --rm -it browshvim
```

It hangs on the "Waiting for Firefox to connect..." screen. - Actually, it seems to display that screen for the normal duration, then it redraws that screen and stays stuck there.

I'm using Kitty 0.14.4 and tests for 256 color support are working

**Comment by tombh** [comment]
Maybe this should be taken up on that PR? Also it would be helpful to get the logs from `browsh --debug`, which is a bit tricky when running in Docker, probably better to do `docker run --rm -it browshvim bash` and then run `browsh --debug` so you can access "debug.log" before the container gets removed.

**Comment by InconsolableCellist** [comment]
I left a comment on that issue but it was marked as closed so I figured it wouldn't have gotten seen. I could move discussion there if there's a way to re-open it. 

Thanks for the commands. Here's the full contents of debug.log after running ./browsh in the container and waiting about a minute:

```
Looking in /app/.config/browsh/config.toml for config.
Attempting to connect to Firefox Marionette
Starting Firefox in headless mode
Using Firefox at: /app/bin/firefox/firefox
Using default profile at: /app/.config/browsh/firefox_profile
Sending `WebDriver:NewSession` to Firefox Marionette
Sending `Addon:Install` to Firefox Marionette
Starting Browsh CLI client
Starting websocket server...
Webextension not connected. Message not sent: /tty_size,137,41
FF-MRNT: 50:{"applicationType":"gecko","marionetteProtocol":3}
Webextension not connected. Message not sent: /stdin,{"char":"\r","key":13,"mod":0}
Webextension not connected. Message not sent: /stdin,{"char":"\r","key":13,"mod":0}
Webextension not connected. Message not sent: /stdin,{"char":"\r","key":13,"mod":0}
Webextension not connected. Message not sent: /stdin,{"char":"\r","key":13,"mod":0}
Webextension not connected. Message not sent: /stdin,{"char":"\r","key":13,"mod":0}
Webextension not connected. Message not sent: /stdin,{"char":"\r","key":13,"mod":0}
Webextension not connected. Message not sent: /stdin,{"char":"\r","key":13,"mod":0}
```

**Comment by InconsolableCellist** [comment]
Something strange happened. I tailed the debug.log from another bash instance and sent a ton of key commands. Either coincidentally or due to an actual memory leak my computer started swapping and I had to reboot it. When doing the steps to reproduce this now, of course, it suddenly works!

I suppose I'll close this then, and perhaps because I'm the reporter I can re-open it if it happens again. Thanks
